1611477989869	Sync.LogManager	DEBUG	Flushing file log
1611477989878	Sync.LogManager	DEBUG	Log cleanup threshold time: 1610613989878
1611477989933	Sync.LogManager	DEBUG	Done deleting files.
1611478589865	Sync.Service	DEBUG	User-Agent: Firefox/84.0.2 (Linux x86_64) FxSync/1.86.0.20210107154745.desktop
1611478589865	Sync.Service	INFO	Starting sync at 2021-01-24 14:26:29 in browser session kudRxEAIjpN6
1611478589866	Sync.Service	DEBUG	In sync: should login.
1611478589866	Sync.Service	INFO	User logged in successfully - verifying login.
1611478589867	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1611478589868	FirefoxAccounts	DEBUG	getOAuthToken enter
1611478589869	FirefoxAccounts	TRACE	getCachedToken returning cached token
1611478589869	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1611478589869	Sync.BrowserIDManager	DEBUG	unlockAndVerifyAuthState already has (or can fetch) sync keys
1611478589869	Sync.Status	DEBUG	Status.login: error.login.reason.network => success.status_ok
1611478589869	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1611478589870	Sync.Service	DEBUG	Fetching unlocked auth state returned success.status_ok
1611478589872	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1611478591624	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile 304
1611478679910	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/177607653/info/collections failed: [Exception... "The connection has timed out"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "<unknown>"  data: no] No traceback available
1611478679911	Sync.Service	DEBUG	verifyLogin failed: [Exception... "The connection has timed out"  nsresult: "0x804b000e (NS_ERROR_NET_TIMEOUT)"  location: "<unknown>"  data: no] No traceback available
1611478679911	Sync.Status	DEBUG	Status.login: success.status_ok => error.login.reason.network
1611478679911	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1611478679911	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1611478679911	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1611478679912	Sync.ErrorHandler	ERROR	Sync encountered a login error
1611478679912	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1611478679914	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1611478679917	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.network(resource://services-sync/service.js:1038:15) JS Stack trace: onNotify@service.js:1038:15
1611478679918	Sync.Service	DEBUG	Not syncing: login returned false.
1611478679919	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
